--- src/configure.orig	2025-03-05 17:45:36.786287732 +0100
+++ src/configure	2025-03-05 17:44:47.104932984 +0100
@@ -55425,13 +55425,15 @@ sysv4 | sysv4.3*)
     ;;
   esac
   ;;
-
 tpf*)
   lt_cv_deplibs_check_method=pass_all
   ;;
 os2*)
   lt_cv_deplibs_check_method=pass_all
   ;;
+genode*)
+  lt_cv_deplibs_check_method=pass_all
+  ;;
 esac
 
 fi
@@ -61048,6 +61060,13 @@ uts4*)
   shlibpath_var=LD_LIBRARY_PATH
   ;;
 
+genode*)
+  dynamic_linker="Genode ld.lib.so"
+  shrext_cmds=.lib.so
+  libname_spec='$name'
+  library_names_spec='$libname$shared_ext'
+  ;;
+
 *)
   dynamic_linker=no
   ;;
@@ -65013,6 +65038,13 @@ uts4*)
   shlibpath_var=LD_LIBRARY_PATH
   ;;
 
+genode*)
+  dynamic_linker="Genode ld.lib.so"
+  shrext_cmds=.lib.so
+  libname_spec='$name'
+  library_names_spec='$libname$shared_ext'
+  #dynamic_linker=yes
+  ;;
 *)
   dynamic_linker=no
   ;;
@@ -71563,7 +71595,8 @@ lt_prog_compiler_wl_CXX='`$ECHO "$lt_pro
 lt_prog_compiler_static_CXX='`$ECHO "$lt_prog_compiler_static_CXX" | $SED "$delay_single_quote_subst"`'
 lt_cv_prog_compiler_c_o_CXX='`$ECHO "$lt_cv_prog_compiler_c_o_CXX" | $SED "$delay_single_quote_subst"`'
 archive_cmds_need_lc_CXX='`$ECHO "$archive_cmds_need_lc_CXX" | $SED "$delay_single_quote_subst"`'
-enable_shared_with_static_runtimes_CXX='`$ECHO "$enable_shared_with_static_runtimes_CXX" | $SED "$delay_single_quote_subst"`'
+enable_shared_with_static_runtimes_CXX='`$ECHO
+"$enable_shared_with_static_runtimes_CXX" | $SED "$delay_single_quote_subst"`'
 export_dynamic_flag_spec_CXX='`$ECHO "$export_dynamic_flag_spec_CXX" | $SED "$delay_single_quote_subst"`'
 whole_archive_flag_spec_CXX='`$ECHO "$whole_archive_flag_spec_CXX" | $SED "$delay_single_quote_subst"`'
 compiler_needs_object_CXX='`$ECHO "$compiler_needs_object_CXX" | $SED "$delay_single_quote_subst"`'
